admin/js/settings.js

```diff
@@ -6,12 +6,16 @@
  * @subpackage Rotaract_Appointments/admin/js
  */
 
+/* globals lcData */
+
 lcSelectInit();
 addEventListeners();
 
-/** Initialize LC-select field. */
+/**
+ * Initializes LC-select targeting the select field.
+ */
 function lcSelectInit() {
-	new lc_select(
+	lc_select(
```

Comment:
Did you that? As I understood we need to to call the constructor here...

Reply:
Change was motivated by <https://jshint.com/docs/options/#nonew>